scripts/st2bootstrap-el7.sh

```diff
@@ -0,0 +1,183 @@
+  # List the actions from a 'core' pack
+  st2 action list --pack=core
+  
+  # Run a local shell command
+  st2 run core.local -- date -R
+  
+  # See the execution results
+  st2 execution list
+  
+  # Fire a remote comand via SSH (Requires passwordless SSH)
+  st2 run core.remote hosts='localhost' -- uname -a
+  
+  # Install a pack
+  st2 run packs.install packs=st2
+}
+
+install_st2mistral_depdendencies() {
+  sudo yum -y install postgresql-server postgresql-contrib postgresql-devel
+  
+  # Setup postgresql at a first time
+  sudo postgresql-setup initdb
+
+  # Make localhost connections to use an MD5-encrypted password for authentication
+  sudo sed -i "s/\(host.*all.*all.*127.0.0.1\/32.*\)ident/\1md5/" /var/lib/pgsql/data/pg_hba.conf
+  sudo sed -i "s/\(host.*all.*all.*::1\/128.*\)ident/\1md5/" /var/lib/pgsql/data/pg_hba.conf
+
+  # Start PostgreSQL service
+  sudo systemctl start postgresql
+
+  cat << EHD | sudo -u postgres psql
+CREATE ROLE mistral WITH CREATEDB LOGIN ENCRYPTED PASSWORD 'StackStorm';
+CREATE DATABASE mistral OWNER mistral;
+EHD
+}
+
+install_st2mistral() {
+  # install mistral
+  sudo yum -yq install st2mistral
+  
+  # Setup Mistral DB tables, etc.
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
+  # Register mistral actions
+  /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+
+  # start mistral
+  sudo systemctl start mistral
+}
+
+verify_st2mistral() {
+  mistral --version
+
+  sudo cp -rf /usr/share/doc/st2/examples /opt/stackstorm/packs
+  st2ctl reload
+  
+  # run mistral examples
+  st2 run examples.mistral_examples
+}
+
+install_st2web() {
+  # Install st2web and nginx
+  sudo yum install -y st2web nginx
+  
+  # Generate self-signed certificate or place your existing certificate under /etc/ssl/st2
+  sudo mkdir -p /etc/ssl/st2
+  
+  sudo openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
+  -days 365 -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information Technology/CN=$(hostname)"
+  
+  # Copy and enable StackStorm's supplied config file
+  sudo cp /usr/share/doc/st2/conf/nginx/st2.conf /etc/nginx/conf.d/
+  
+  # Disable default_server configuration in existing /etc/nginx/nginx.conf
+  sudo sed -i 's/default_server//g' /etc/nginx/nginx.conf
+  
+  sudo systemctl restart nginx
+}
+
+verify_st2web() {
+  # dumb check that st2web index.html works
+  curl --insecure -s "https://127.0.0.1/" | grep -q "stackstorm"
+
+  # st2auth check
+  curl -X OPTIONS -I --insecure https://127.0.0.1/auth/ | grep -q 'St2-Api-Key'
+
+  # st2api check
+  curl -X OPTIONS -I --insecure https://127.0.0.1/api/ | grep -q 'St2-Api-Key'
+}
+
+
+adjust_selinux_policies
+
+install_st2_dependencies
+install_st2
+configure_st2_user
+configure_st2_authentication
+verify_st2
+
+install_st2mistral_depdendencies
+install_st2mistral
+verify_st2mistral
+
+install_st2web
+verify_st2web
```

Comment:
I am not sure why you went with a different approach than ubuntu? https://github.com/StackStorm/st2-packages/blob/master/scripts/st2bootstrap-deb.sh#L119

It would be better to be consistent with the steps in ubuntu. Otherwise, we'd have maintenance nightmares. 

cc: @DoriftoShoes